app/views/baskets/index.html.erb

```diff
@@ -1,2 +1,49 @@
-<h1>Baskets#index</h1>
-<p>Find me in app/views/baskets/index.html.erb</p>
+<% @cart.items.each do |item| %>
+<tr>
+<td>
+<%= image_tag item.pic , :alt => "#{item.title}" %>
+</td>
+<td>
+<%= link_to "#{item.title}" , store_path(item.product_id) %>
+</td>
+<td>
+<%= item.unit_price %>
+</td>
+<td>
+<%= item.quantity %>
+</td>
+<td>
+<%= item.total_price %>
+</td>
+<% end %>
+</tr>
+
+
+<div class="form-popup" id="popup-Form">
+  <form action="/action_page.php" class="form-container">
+    <h2>Veuillez vous connecter</h2>
+    <label for="email">
+      <strong>E-mail</strong>
+    </label>
+    <input type="text" placeholder="Votre Email" name="email" required />
+    <label for="psw">
+      <strong>Mot de passe</strong>
+    </label>
+    <input type="password" placeholder="Votre mot de passe" name="psw" required />
+    <button type="submit" class="btn">Se connecter</button>
+    <button type="submit" class="btn cancel" onclick="closeForm()">Quittez votre panier</button>
+  </form>
+</div>
+
+
+
+
+
+
+
+<% @product.each do product %>
+<%= product.name %>
+<%= product.pricing %>
+<%= product.quantity %>
+<%= product.category %>
+ <% end %>
```

Comment:
We are in the baskets_index, we shouldnt have a @product instance

we should have a @baskets storing all baslets instances and iterate over them to display each one's basket_items and products